src/main/java/com/toptrumps/db/GameDAO.java

```diff
@@ -0,0 +1,6 @@
+package com.toptrumps.db;
+import com.toptrumps.core.statistics.GameStateCollector;
+
+public interface GameDAO {
```

Comment:
It can be package-private

Reply:
"package-private (often just called package) means that other members of the same package have access to the item. package-private is the default access modifier and does not have a keyword, because package is used to specify the package for a class or interface."